File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/ClientRecordRequest.java
package kz.greetgo.sandbox.controller.model;

import kz.greetgo.sandbox.controller.model.ColumnSortType;

public class ClientRecordRequest {
  public long clientRecordCountToSkip;
  public long clientRecordCount;
  public ColumnSortType columnSortType;
  public boolean sortAscend;
  public String nameFilter;
}

File: sandbox.client/front/ts/model/ClientRecordRequest.ts
import {ColumnSortType} from "./ColumnSortType";

export class ClientRecordRequest {
  public clientRecordCountToSkip: number/*long*/;
  public clientRecordCount: number/*long*/;
  public columnSortType: ColumnSortType;
  public sortAscend: boolean;
  public nameFilter: string;
}
